File: StandOrganizer.Api/StandOrganizer.DAL/Models/Service.cs
using System;

namespace StandOrganizer.DAL.Models
{
     // Dont use object for assistant and publisher to avoid saving them to separate table. 
    public class Service : BaseEntity
    {
        public DateTimeOffset StartTime { get; set; }

        public DateTimeOffset EndTime { get; set; }

        public string PublisherName { get; set; }

        public string PublisherPhone { get; set; }

        public string AssistantName { get; set; }

        public string AssistantPhone { get; set; }

        public virtual Stand Stand { get; set; }
    }
}

File: StandOrganizer.App/src/types/Service.js
import {calcAccurateHours} from '../utils/timeHelper';

export default class Service {
  constructor(id, startTime, endTime, whitness, assistant) {
    this.id = id;
    this.startTime = startTime;
    this.endTime = endTime;
    this.whitness = whitness;
    this.assistant = assistant;

    this.duration = calcAccurateHours(endTime) - calcAccurateHours(startTime);
  }
}
